dex/testing/firo/electrum.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# https://github.com/spesmilo/electrum/issues/7833 (fixed in 4.3)
+# possible alt workaround: python3 -m pip install "protobuf>=3.12,<4"
```

Comment:
This script sets up an electrum client? Can you add a comment like you have with the server. Also, can you state the dependencies? I only need need python3?

You could also add a readme like https://github.com/decred/dcrdex/blob/master/dex/testing/dcr/README.md

Reply:
README's done

Reply:
_electrum(x) and  scripts - code comments done_